sinks/librato.py

```diff
@@ -288,13 +285,16 @@ def add_measure(self, key, value, time):
             k = name
 
         if k not in self.measurements:
-            self.measurements[k] = {
-                'name': name,
-                'tags' : tags,
-                'time' : ts
-            }
+            self.measurements[k] = []
```

Comment:
Changing `measurements[k]` to be an array of measurements so that we can have multiple tagsets for metric `k`.